FirebaseMessaging/Tests/UnitTests/FIRMessagingAPNSInfoTest.m

```diff
@@ -72,15 +72,20 @@ - (void)testAPNSInfoCreationWithInvalidSandboxFormat {
 - (void)testAPNSInfoEncodingAndDecoding {
   NSDictionary *validDictionary = @{
     kFIRMessagingTokenOptionsAPNSKey : [@"tokenData" dataUsingEncoding:NSUTF8StringEncoding],
-    kFIRMessagingTokenOptionsAPNSIsSandboxKey : @"sandboxValueAsString"
```

Comment:
This was a bug before, caught by the transition to NSSecureCoding. In the old implementation the constructor return nil and all the tests compared nil to nil.